src/dust/dust.cpp

```diff
@@ -97,8 +103,17 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   for (int n = 0; n < nspecies; ++n)
     dustids.push_back(n);
 
+  bool hst_out_d2g = pin->GetOrAddBoolean("dust", "hst_out_d2g", false);
+  params.Add("hst_out_d2g", hst_out_d2g);
+
+  // coagulation flag
+  bool enable_dust_coagulation = pin->GetOrAddBoolean("physics", "coagulation", false);
+
   // Dust sizes
-  const auto size_dist = pin->GetOrAddString("dust", "size_input", "direct");
+  auto size_dist = pin->GetOrAddString("dust", "size_input", "direct");
+  if (enable_dust_coagulation) size_dist = "logspace";
```

Comment:
I don't see why coagulation requires `logspace`? I would remove this

Reply:
see my comment above